docs/pyproject.toml

```diff
@@ -6,11 +6,14 @@ authors = []
 dependencies = [
     "pydata-sphinx-theme>=0.16.0",
     "sphinx>=8.0.2",
-    "pyvortex",
+    "vortex-array",
```

Comment:
this has been wrong since we renamed the package but was fine because `rye` was installing the workspace members